unix_integration/resolver/src/bin/kanidm-unix.rs

```diff
@@ -228,7 +229,11 @@ async fn main() -> ExitCode {
             } else {
                 match call_daemon(cfg.sock_path.as_str(), req, cfg.unix_sock_timeout).await {
                     Ok(r) => match r {
-                        ClientResponse::Ok => println!("working!"),
+                        ClientResponse::ProviderStatus(results) => {
+                            for provider in results {
+                                println!("{}: {}", provider.name, provider.online);
```

Comment:
```suggestion
                                println!("{}: {}", provider.name, "online" if provider.online else "offline");
```

Reply:
I actually didn't know you could do that syntax lol.